keystore/keystore.go

```diff
@@ -27,28 +30,38 @@ type Helper struct {
 
 // Execute ...
 func Execute(cmdSlice []string) error {
-	prinatableCmd := cmdex.PrintableCommandArgs(false, cmdSlice)
-	log.Detail("=> %s", prinatableCmd)
+	prinatableCmd := command.PrintableCommandArgs(false, cmdSlice)
+	log.Printf("=> %s", prinatableCmd)
 	fmt.Println("")
 
-	cmd, err := cmdex.NewCommandFromSlice(cmdSlice)
+	cmd, err := command.NewFromSlice(cmdSlice)
 	if err != nil {
 		return fmt.Errorf("Failed to create command, error: %s", err)
 	}
 
```

Comment:
you should `command.PrintableCommand()` here, instead of doing it before the command would be created (line 32: `log.Printf("=> %s", prinatableCmd)` [it is just code style])